static/js/compose.js

```diff
@@ -672,6 +673,7 @@ export function initialize() {
         e.stopPropagation();
 
         clear_preview_area();
+        autosize.update($("#compose-textarea"));
```

Comment:
Should this be in `clear_preview_area` instead?  I think that would be more correct, since in particular it'd work with any future keyboard shortcut to render the preview.

Reply:
Makes sense.